Concrete_article_data.py

```diff
@@ -0,0 +1,54 @@
+# -*-encoding: utf-8 -
+
+
+import numpy as np
+import pandas as pd
+from sklearn import gaussian_process
+import matplotlib.pyplot as plt
+
+if __name__ == '__main__':
+
+    df = pd.read_excel('C:/users/Nicolas/PycharmProjects/Concrete_Data.xls', header=0).dropna()
```

Comment:
Would be good to have this line in a `try... except...` block as reading a file is an operation that can generate exceptions...